app-react/src/components/Header.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import stickyNote from '../images/sticky_note-white.svg';
+
+const HeaderStyled = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 2;
+  position: relative;
+  background: var(--main-b-color);
+  padding: 15px 20px;
+  box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.07), 0 3px 10px 0 rgba(0, 0, 0, 0.07);
+  color: white;
+  min-height: 50px;
+  max-height: 50px;
+`;
+
+const HeaderTitle = styled.div`
+  display: flex;
+  h1 {
+    font-size: 28px;
+    margin: 0;
+  }
+  img {
+    width: 24px;
+    margin-left: 6px;
+    margin-bottom: -6px;
+  }
+`;
+
+const ProfileContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const SignoutButton = styled.button`
+  background: white;
+  color: var(--main-b-color);
+  border-radius: 3px;
+  font-size: 14px;
+  font-weight: 700;
+`;
+
+const AvatarImg = styled.img`
+  height: 50px;
+  width: 50px;
+  border-radius: 53px;
+  margin-left: 20px;
+  box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.07), 0 3px 10px 0 rgba(0, 0, 0, 0.07);
+`;
+
+const Header = ({ profileVisible, avatarImgPath }) => {
+  return (
+    <HeaderStyled class='header'>
+      <HeaderTitle class='header-title'>
+        <h1>dev-jot</h1>
+        <img alt='note icon' src={stickyNote} />
+      </HeaderTitle>
+      {profileVisible ? (
+        <ProfileContainer class='profile'>
+          <div hidden class='sign-out'>
+            <SignoutButton class='button'>Sign out</SignoutButton>

```

Comment:
You weren't to know this, but this sign-in/out button is now redundant. Initially the app used Firebase authentication – hence the button. I ended up removing Firebase because contributors were having trouble with set up. No need to remove this now. We can always do this in a follow up ticket 😄